src/styles/Company.scss

```diff
@@ -0,0 +1,58 @@
+body {
+  background-color: rgb(215,215,215);
+}
+
+h1 {
+  font-size: 40px;
+  font-weight: 800;
+}
+
+.page-header {
+  width: 150px;
+  margin-left: 20px;
+  background-color: rgba($color: #e9c6c6, $alpha: 0.75);
+}
+
+.company {
+  margin: 80px;
+  text-align: center;
+}
+
+.company-logo {
+  width: 200px;
+}
+
+.company-snapshot {
+  text-align: center;
+  background-color: rgba($color: #776c6c, $alpha: 0.75);
+  padding:30px;
+}
+
+.demographics {
+  padding: 15px;
```

Comment:
It seems like `.demographics`, `.labor-practices` etc. all have this common property of `padding: 15px;`
This makes me think they all have a common styling as companies info categories. To keep code DRY, esp as we may scale since there will probably be more styling added, I'd recommend making one class where you have this padding, and then look at each category with its `id` (not class, since there is only one of each on the page).